### PR TITLE
Fixed context issue

### PR DESCRIPTION
**Pull request by anshulvermapatel**
### Which issue this PR addresses:

Fixes a issue where a different context being used while doing get token call to azure than the one sent to individual db clients.

### CODE REVIEW

File: pkg/database/cosmosdb/zz_generated_authorizer.go [hunk 1]
```diff
@@ -15,14 +16,14 @@ import (
 )
 
 type Authorizer interface {
-	Authorize(*http.Request, string, string) error
+	Authorize(context.Context , *http.Request, string, string) error
```
Comment (niontive) [inline pkg/database/cosmosdb/zz_generated_authorizer.go:19]: Nit: `Authorize(context.Context, *http.Request, string, string) error`

Extra space between `Context` and `.`

Comment (anshulvermapatel) [reply pkg/database/cosmosdb/zz_generated_authorizer.go]: Done.

**Review by tiguelu — APPROVED** [review]
LGTM

**Review by mociarain — APPROVED** [review]

**Pull request merged**